src/customer-view/components/CustomerView.css

```diff
@@ -0,0 +1,89 @@
+/* variables */
+:root {
+  --blurred-border: 2px solid rgb(220, 220, 220);
+}
+
+/* general style */
+* {
+  box-sizing: border-box;
+  font-family: Calibri, sans-serif;
+  font-size: 17px;
+}
+
+/* composable style */
+.button {
```

Comment:
`.button` is bad, try more specific classname, that different from global namespace
`wrapper_component-classname__button`